### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by cjpatton**
Based on #23 (merge that first).

Miscellaneous things:

* Fix minor issues with APIs
* Rename `prio3_prepare_start()`
* Triage remaining TODOs
* Update READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,59 @@
...
+under active development in the PPM working group of the IETF.
 
-[miniflare](https://miniflare.dev/) is used to test integration of Daphne-Worker
-with Daphne. These tests can be run from Docker:
+This software is intended to support experimental DAP deployments and is not yet
+suitable for use in production. Daphne will evolve along with the DAP draft:
+Backwards compatibility with previous drafts won't be guaranteed until the draft
+itself begins to stabilize. API-breaking changes between releases should also be
+expected.
+
```
Comment (armfazh) [inline README.md:12]: maybe just mention how version number is related to the draft.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,59 @@
...
+[miniflare](https://miniflare.dev/) to mock the Cloudflare Workers platform.
+These can via docker-compose:
 
 ```
 docker-compose up --build --abort-on-container-exit --exit-code-from test
 ```
 
-## Development environment
+However for development it's often more convenient to run miniflare directly.
+Prrequisites:
```
Comment (armfazh) [inline README.md:42]:
```suggestion
Prerequisites:
```

File: daphne/src/lib.rs [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
-//!  - Prio3Aes128Histogram
-//!  - Prio3Aes128Sum
+//! This crate implements the core protocol logic for the Distributed Aggregation Protocol
+//! ([DAP](https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap)) standard under development in the
+//! PPM working group of the IETF. See [`VdafConfig`] for a listing of supported
+//! [VDAFs](https://github.com/cfrg/draft-irtf-cfrg-vdaf).
 //!
-//! Additional Prio3 variants can be added easily as need.
+//! Daphne does not provide the complete, end-to-end functionality of any party in the protocol.
+//! Instead, it defines traits for the functionalities that a concrete instantantiation of the
```
Comment (armfazh) [inline daphne/src/lib.rs:10]:
```suggestion
//! Instead, it defines traits for the functionalities that a concrete instantiation of the
```

**Review by armfazh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cjpatton) [reply README.md:12]: I don't want to promise anything here until we get to draft-01.

**Pull request merged**